src/sdp/media/ersip_sdp_attr_rtpmap.erl

```diff
@@ -36,21 +31,23 @@
     type           :: payload_type(),
     encoding_name  :: encoding_name(),
     clock_rate     :: clock_rate(),
-    params         :: encoding_params() 
+    params         :: encoding_params()
 }).
 
--type payload_type()     :: binary().
+-type payload_type()     :: pos_integer().
 -type encoding_name()    :: binary().
 -type clock_rate()       :: pos_integer().
 -type encoding_params()  :: pos_integer() | undefined.
 -type rtpmap()           :: #rtpmap{}.
--type parse_result()     :: parse_result(rtpmap()).
+-type parse_result()     :: {ok, rtpmap()} |
+                            {error, {invalid_rtpmap_candidate, term()}}.
```

Comment:
Why do you call it "candidate"? rtpmap is just "attribute". It would prefer just `invalid_rtpmap`